src/modules/automod/message.js

```diff
@@ -10,35 +10,20 @@ import config from '../../config';
 /**
  * Checks if a message contains a swear
  * @param message The message to check
- * @return {RegExpMatchArray} any match with the blacklist
+ * @return {string} any match with the blacklist
  */
 function containsSwear(message) {
-    const content = message.content
-        .toLowerCase()
-        .replaceAll(/[0ö]/g, 'o')
-        .replaceAll(/[1!|]/g, 'i')
-        .replaceAll(/3/g, 'e')
-        .replaceAll(/[4@ä]/g, 'a')
-        .replaceAll(/[5$]/g, 's')
-        .replaceAll(/[7+]/g, 't')
-        .replaceAll(/8/g, 'b')
-        .replaceAll(/\(/g, 'c')
-        .replaceAll(/ü/g, 'u')
-        .replaceAll(/[^a-z ]/g, '');
-    const regex = Blacklist.swearRegex;
-    return content.match(regex);
+    const words = message.content.toLowerCase().split(' ');
```

Comment:
splitting by `/(?:\b|\s)+/g` might be smarter.
![435F7E2F-F93A-4CA8-8812-89A1B4472471](https://user-images.githubusercontent.com/33164598/160288380-27b2a85f-8c73-40aa-9aa9-93b11a7ad8dc.jpeg)